style.css

```diff
@@ -0,0 +1,113 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+
+body {
+  min-height: 100vh;
+  width: 100%;
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  overflow-x: hidden;
+  overflow-y: hidden;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  background: #1c1a19;
+  background-image: url(./images/illustration.png);
+  background-position: bottom -40px right;
+  background-repeat: no-repeat;
+  background-size: auto;
+  min-height: 100vh;
+}
+
+.nav-bar {
+  background: #3c3a39;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 20px;
+  height: 13.7vh;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.nav-bar a {
+  text-decoration: none;
+  color: #fff;
+  font-size: 25px;
+}
+
+.nav-bar img {
+  height: 15px;
+}
```

Comment:
- [ OPTIONAL] This section does not have the same margins (top, bottom, left, right) defined in the original design, Kindly use the same value, you can check the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)